.github/workflows/pre-release.yml

```diff
@@ -55,4 +72,4 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: |
-          aws s3 sync bin s3://botpress-dev-bins/studio/${{ steps.info.outputs.branch_sanitized }}
+          aws s3 sync bin s3://botpress-dev-bins/studio/${GITHUB_HEAD_REF}
```

Comment:
Can we pass this reference to extract_info instead? The reason we use "branch_sanitized" is because we can't use the branch name directly, branches with a slash cause problem. This work for simple branches, but there's branches like "features/DEV-1240". That action converts any non-word with an underscore, so we can safely handle it, and the downloader also uses the same logic